src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+}
+
+.card {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__product-name {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-family: $font-family;
+    color: $main-accent-color;
+    width: 100%;
```

Comment:
The variable `$main-accent-color` should be defined in your `veriables.scss` file or imported correctly if it's defined elsewhere. Ensure that `$main-accent-color` is available in this context.